backend/inspirehep/search/facets.py

```diff
@@ -218,15 +218,25 @@ def records_hep(order=None):
 
 
 def citation_summary():
-    excluded_filters = ["citeable", "refereed", "citation_count"]
+    excluded_filters = [
+        "citeable",
+        "refereed",
+        "citation_count",
+        "citation_count_without_self_citations",
+    ]
     filters = get_filters_without_excluded(hep_filters(), excluded_filters)
-    map_script = """
-        if (doc.refereed.length >0 && doc.refereed[0]) {
-            state.citations_refereed.add(doc.citation_count[0])
-        } else {
-            state.citations_non_refereed.add(doc.citation_count[0])
-        }
-    """
+    if request.values.get("exclude-self-citations", False):
+        field = "citation_count_without_self_citations"
+    else:
+        field = "citation_count"
```

Comment:
```suggestion
    exclude_sef_citations = request.values.get("exclude-self-citations", False)
    field = "citation_count_without_self_citations" if exclude_self_citations else "citation_count"
```

Reply:
Sorry but what's the difference except that your code is a little less readable? ;)

Reply:
That's why it says "Suggested change", it's a suggestion :)

Reply:
:+1: